kernel/memory/src/paging/mapper.rs

```diff
@@ -877,7 +891,9 @@ impl MappedPages {
             return Err("MappedPages::as_slice_mut(): requested slice length and byte_offset would not fit within the MappedPages bounds");
         }
 
-        // SAFE: we guarantee the bounds and lifetime are within that of this MappedPages object
+        // SAFETY:
+        // ✅ same as for `MappedPages::as_slice()`, plus:
+        // ✅ The underlying memory is not accessible through any other pointer, as we require a `&mut self` above.
```

Comment:
We've also checked that the underlying memory is writable. That's mentioned in `BorrowedMappedPages::deref_mut`.

Reply:
yup, thought about it and forgot to commit it. thanks!